src/articles/articles.controller.ts

```diff
@@ -48,4 +69,41 @@ export class ArticlesController {
   remove(@Param('id') id: string) {
     return this.articlesService.remove(+id);
   }
+
+  @Post(':articleId/comment')
+  createComment(
+    @CurrentUser() user: User,
+    @Body() comments: CreateCommentDto,
+    @Param('articleId') articleId: number,
+  ) {
+    const comment = this.commentService.create(user, comments, articleId);
+    return comment;
```

Comment:
`return this.commentService.create(user, comments, articleId);`
Nếu không cần dùng làm gì thì ko nên gán biến